engines/director/resource.cpp

```diff
@@ -435,6 +435,7 @@ void Window::loadStartMovieXLibs() {
 	if (strcmp(g_director->getGameId(), "warlock") == 0 && g_director->getPlatform() != Common::kPlatformWindows) {
 		g_lingo->openXLib("FPlayXObj", kXObj);
 	}
+	g_lingo->openXLib("FileIO", kXObj);
```

Comment:
Why did you decide to put it by default?

Reply:
That was due to one Director movie that relied on being launched from the main projector that lacked any embedded movie. I realise that the functionality is there but I'll have to look at it again as I believe it wasn't working when I checked.